idaes/core/surrogate/tests/test_pysmo_surrogate.py

```diff
@@ -1633,6 +1633,74 @@ def test_populate_block_multisurrogate_poly_userdef(self, pysmo_surr4):
             "outputs[z2] - (-12.523574144487087 - 2.1308935361219556*inputs[x1] + 4.1308935361216435*inputs[x2] + 3.6347869158959156e-12*(inputs[x1]/inputs[x2]))"
         )
 
+    @pytest.mark.parametrize(
+        "confidence_dict", [{0.99: 3.2498355440153697}, {0.90: 1.8331129326536335}]
+    )
+    @pytest.mark.unit
+    def test_confint_default(self, confidence_dict):
+        training_data = {
+            "x1": [1, 2, 3, 4, 5],
+            "x2": [5, 6, 7, 8, 9],
+            "z1": [10, 20, 30, 40, 50],
+            "z2": [6, 8, 10, 12, 14],
+        }
+        training_data = pd.DataFrame(training_data)
+        validation_data = {
+            "x1": [1, 2, 3, 4],
+            "x2": [5, 6, 7, 8],
+            "z1": [10, 20, 30, 40],
+            "z2": [6, 8, 10, 12],
+        }
+        validation_data = pd.DataFrame(validation_data)
+        input_labels = ["x1", "x2"]
+        output_labels = ["z1", "z2"]
+        bnds = {"x1": (0, 5), "x2": (0, 10)}
+
+        pysmo_trainer = PysmoPolyTrainer(
+            input_labels=input_labels,
+            output_labels=output_labels,
+            input_bounds=bnds,
+            training_dataframe=training_data,
+            validation_dataframe=validation_data,
+            maximum_polynomial_order=1,
+            multinomials=True,
+            number_of_crossvalidations=3,
+        )
+
+        a2_poly = pysmo_trainer.train_surrogate()
+        for k in confidence_dict.keys():
+            confidence = k
+            tval = confidence_dict[k]
+
+        output = pysmo_trainer.get_confidence_intervals(a2_poly, confidence)
```

Comment:
What does `output` look like, and what is this test checking? I remember a confidence interval function from an old ALAMO method calculated determined the upper and lower confidence functions using the coefficients of the trained model.

Reply:
Output is a dictionary containing the upper and lower confidence functions using the coefficients of the trained model. The test is checking:

1. That the dictionary contains the rght number of elements (i.e. the number of output variables)
2. That for each of the output variables, the right upper and lower confidence bounds are calculated.